views/dapp/views/dex-pool-details/components/liquidity-details-card/index.tsx

```diff
@@ -9,66 +10,71 @@ import LiquidityDetailsCardLine from './liquidity-details-card-line';
 const LiquidityDetailsCard: FC<LiquidityDetailsCardProps> = ({
   lines,
   isStable,
-}) => (
-  <Box
-    p="L"
-    width="100%"
-    display="flex"
-    bg="foreground"
-    borderRadius="M"
-    flexDirection="column"
-  >
-    <Typography
-      width="100%"
-      fontSize="S"
-      variant="normal"
-      textTransform="uppercase"
-    >
-      Liquidity
-    </Typography>
+}) => {
+  const t = useTranslations('dex-pool-pair-address');
+  const tCommon = useTranslations('common');
```

Comment:
cant u useTranslation without passing a key. 

Rather have t('dex-pool-pair-address.something') and t('common.something')